assets/csi_controller_deployment.yaml

```diff
@@ -2,7 +2,7 @@ kind: Deployment
 apiVersion: apps/v1
 metadata:
   name: csi-snapshot-controller
-  namespace: openshift-cluster-storage-operator
+  namespace: ${NAMESPACE}
```

Comment:
What do you think about using placeholders that specifie the cluster? Something like `${CONTROLPLANE_NAMESPACE}` and `${GUEST_NAMESPACE}`.

In my PR I only have `${NAMESPACE}` too, but now that I see it, the suggestion above will make things clearer..

Reply:
+1 for `${CONTROLPLANE_NAMESPACE}`, but the guest namespace is the same in standalone OCP and in HyperShift, it does not need to be replaced at all.